mne/decoding/csp.py

```diff
@@ -222,7 +236,8 @@ def transform(self, X, epochs_data=None):
         X = np.asarray([np.dot(pick_filters, epoch) for epoch in X])
 
         # compute features (mean band power)
-        X = (X ** 2).mean(axis=-1)
+        if self.transform_into == 'average_power':
+            X = (X ** 2).mean(axis=-1)
         if self.log:
```

Comment:
`log` is True by default, that will cause issue if we forget to set it to False when `transform_into` is `csp_space`.

should we add another mode like `average_log_power` (that will be default)